chapters/derivationofstream.tex

```diff
@@ -206,7 +206,7 @@ \subsection{Ideal Splitting Junction for Uni-Directional Flow - Connection of 3
 \end{equation*}
 
 \lstinline!inStream! cannot be evaluated for a connector, on which the mass flow rate has to be negative by definition.
-The reason is that the value is arbitrary, which is why it is defined as follows.
+Since the value is arbitrary, it is defined as follows
```

Comment:
Suggesting just a minor change here, and will propose a larger change in the review comment:
```suggestion
This is not a problem since there is no requirement on the result of \lstinline!inStream! in this case, and the following result is selected instead of the undefined division result:
```